chart/testdata/frobnitz/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Do we need another copy of the Apache license? I don't think so. The one at the root covers the entire contents of the repository.